README.md

```diff
@@ -148,73 +148,29 @@ _Note that options were changed in version 0.5.3_
 For example these commands:
 
 ```python
-worldengine world -s 4 -n an_example -p 2048 -q 25 -x 2048 -y 2048
+worldengine -b -s 4 -n an_example -q 25 -x 2048 -y 2048
 ```
 
 Produce this output
 
 ```
-Worldengine - world generator
+Worldengine - a world generator (v. 0.18.0)
 -----------------------
- seed              : 4
- name              : seed3
- width             : 2048
- height            : 2048
- plates resolution : 2048
- number of plates  : 25
- operation         : world generation
- step              : full
+ operation              : world generation
+ seed                 : 4
+ name                 : an_example
+ width                : 2048
+ height               : 2048
+ number of plates     : 25
+ world format         : protobuf
+ black and white maps : False
+ step                 : full
+ greyscale heightmap  : False
+ rivers map           : False
+ scatter plot         : False
+ fade borders         : True
 
 starting (it could take a few minutes) ...
-...plates simulated
```

Comment:
Having an exact copy of the verbose-output in the README is kind of ugly when it comes to changing or adding messages (the README will probably be overlooked). I think this should be left out - as an alternative an image of the final biomes would convey roughly the same information and provide some eye-candy on top of it.